linker/Tests/Mono.Linker.Tests.Cases/TypeForwarding/MissingTargetReference.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+using Mono.Linker.Tests.Cases.Expectations.Metadata;
+
+namespace Mono.Linker.Tests.Cases.TypeForwarding
+{
+	[Define ("IL_ASSEMBLY_AVAILABLE")]
+	[SetupCompileBefore ("TypeForwarderMissingReference.dll", new [] { "Dependencies/TypeForwarderMissingReference.il" })]
+	[SetupLinker ("-p link TypeForwarderMissingReference.dll")]
```

Comment:
Let me give you an example following the pattern you have here

[SetupLinkerOptions("-b true -t false -g false")]

So first, my problem with that.  
* Unless you grok the linker args, that is beyond meaning less. 
* Given that the linker args seem to be just arbitrary letters of the alphabet paired with some functionality, it's very hard to remember what they all do.

On the other hand, if I looked at a test case and saw something like [LinkSymbols].  That quickly communicates that the test case is going to link the symbol files. 

Now back to argument translation, given the above set of arguments, I need to pass the following to the UnityLinker
"--link-symbols --disable-keep-facades --disable-regen-guid"

Giving the example used in this PR "-p link TypeForwarderMissingReference.dll", I'd need to translate that to "-p=link,TypeForwarderMissingReference.dll"

None of this is impossible, but it's going to be tedious and error prone for us to parse whole strings and convert them.

The current test case attribute scheme came to be for a reason.  It let's us avoid arg parsing entirely in our test code and helps cover up what is in my opinion a set of command line arguments that do nothing to communicate their meaning.  This approach admittedly has it's short comings, one being it's very tedious to extend to support for new options. 

We need to find a new approach that meets in the middle.  I think an attribute accepting a raw string of args is too far in the open direction.  It will make our life harder, and aside from the 2-3 people who have memorized the linker args (I'm not one of them) it's not more readable.